Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextBoxMask/TextBoxMaskPage.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.TextBoxMaskPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <Style x:Key="TitleText"
+               TargetType="TextBlock">
+            <Setter Property="Margin" Value="5,20,0,5" />
+            <Setter Property="FontSize" Value="16" />
+            <Setter Property="Foreground" Value="{StaticResource Brush-Grey-03}" />
+            <Setter Property="HorizontalAlignment" Value="Left" />
+        </Style>
+        <Style x:Key="DividingBorder"
+               TargetType="Border">
+            <Setter Property="BorderThickness" Value="0,0,0,1" />
+            <Setter Property="BorderBrush" Value="Black" />
+            <Setter Property="Margin" Value="0,0,0,20" />
+        </Style>
+    </Page.Resources>
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <StackPanel>
+            <TextBlock Style="{StaticResource TitleText}">Text box with Mask 9a9a-a9a*</TextBlock>
+            <Border Style="{StaticResource DividingBorder}" />
+            <TextBox Width="166"
+                     Height="26"
+                     VerticalAlignment="Top"
+                     controls:TextBoxMask.Mask="9a9a-a9a*"
+                     TextWrapping="Wrap" />
+
+            <TextBlock Style="{StaticResource TitleText}">Text box with Mask +1999-9999 and place holder space</TextBlock>
```

Comment:
Won't work best with stack panel and looks better with adding the border, the stack panel needs to be converted to grid and add some margins to the textbox to match the current view.